csharp/custom-resource-cloudfront-invalidate/src/CustomResourceCloudFrontInvalidate/CustomResourceCloudFrontInvalidateStack.cs

```diff
@@ -0,0 +1,66 @@
+using System;
+using Amazon.CDK;
+using Amazon.CDK.AWS.CloudFront;
+using Amazon.CDK.AWS.CloudFront.Origins;
+using Amazon.CDK.AWS.S3;
+using Amazon.CDK.AWS.S3.Deployment;
+using CustomResourceCloudFrontInvalidate.Invalidation;
+using Constructs;
+
+namespace CustomResourceCloudFrontInvalidate
+{
+    public class CustomResourceCloudFrontInvalidateStack : Stack
+    {
+      internal CustomResourceCloudFrontInvalidateStack(
+        Constructs.Construct scope,
+        string id,
+        IStackProps props = null
+      ) : base(scope, id, props)
+      {
+        // Create Example Bucket
+        Bucket bucket = new Bucket(this, $"{id}-bucket", new BucketProps()
+        {
+          BucketName = $"example-bucket-change-me-{DateTime.Now.Ticks.ToString()}"
```

Comment:
This is going to cause a new bucket to be created each time for a synth because it thinks you are wanting to change the name of the bucket.